src/content/structured/components/chips/code.mdx

```diff
@@ -166,79 +114,67 @@ export const snippetsDefault = [
 export const snippetsOutline = [
   {
     language: "Web component",
-    snippet: `<ic-chip label="Small" size="small" appearance="outline">
+    snippet: `<ic-chip label="Americano" size="small" variant="outlined">
```

Comment:
We shouldn't show different sizes here. The section is just about outline, so only show an example of the outline chip.